packages/request/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "noUnusedParameters": true,
     "removeComments": true,
     "sourceMap": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
is this needed?

Reply:
Unfortunately yes. To use environment variables I had to use tsup config file but when I added it I got build errors like:
```
ERROR: Transforming const to the configured target environment ("es5") is not supported yet
```
So I had to bump the target.

Reply:
Reverted.